[GeoMechanicsApplication] Fixed a typo: 'PlainStrain' => 'PlaneStrain'

avdg81 opened the pull request:
**📝 Description**

Fixed a typo in two element type names: 'PlainStrain' => 'PlaneStrain'.

avdg81 committed:
Fixed a typo: 'PlainStrain' => 'PlaneStrain'

rfaasse commented on applications/GeoMechanicsApplication/geo_mechanics_application.cpp:
```diff
@@ -155,8 +155,8 @@ void KratosGeoMechanicsApplication::Register()
 
-    KRATOS_REGISTER_ELEMENT("Geo_UPwLineInterfacePlainStrainElement2Plus2N", mUPwLineInterfacePlaneStrainElement2Plus2N)
-    KRATOS_REGISTER_ELEMENT("Geo_UPwLineInterfacePlainStrainElement3Plus3N", mUPwLineInterfacePlaneStrainElement3Plus3N)
+    KRATOS_REGISTER_ELEMENT("Geo_UPwLineInterfacePlaneStrainElement2Plus2N", mUPwLineInterfacePlaneStrainElement2Plus2N)
```
If we're renaming, maybe we can make it U instead of UPw? Also, could you make sure that the Quay wall model is updated?

avdg81 replied:
Good catch! The line interface element's type name no longer contains the 'Pw' part.

Thank for reminding me to also update the input files of the quay wall model.

rfaasse reviewed (commented):
Just one minor suggestion for this PR

avdg81 committed:
Processed Richard's review comments

Removed the 'Pw' part from the interface element's type name. Also renamed two data members to reflect that.

rfaasse reviewed (approved):
Nice, thanks!
